runelite-client/src/main/java/net/runelite/client/plugins/itemstats/ItemStatOverlay.java

```diff
@@ -182,7 +182,9 @@ private String getChangeString(
 	private String buildStatBonusString(ItemStats s)
 	{
 		final StringBuilder b = new StringBuilder();
-		b.append(getChangeString("Weight", s.getWeight(), true, false));
+		if (config.showWeight()) {
```

Comment:
All brackets should be on a new line, follow our checkstyle: https://github.com/runelite/runelite/wiki/Checkstyle-with-IntelliJ-IDEA